components/LandingPage/BenefitsComponent/Benefit.tsx

```diff
@@ -1,7 +1,7 @@
-import React, { FC } from 'react';
-import { Benefit as BenefitType } from './Benefit.interface';
+import { FC } from 'react';
+import { Benefit as BenefitProps } from './Benefit.interface';
```

Comment:
could you please make the original interface name as 'BenefitProps' instead of renaming here.